package.json

```diff
@@ -24,15 +24,23 @@
   },
   "homepage": "https://github.com/graphcool/graphql-yoga",
   "scripts": {
-    "prepublish": "npm run build",
+    "prepublish": "yarn build",
     "build": "rm -rf dist && tsc -d",
     "lint": "tslint --project tsconfig.json {src,test}/**/*.ts && prettier-check --ignore-path .gitignore {src,.}/{*.ts,*.js}",
     "format": "prettier --write --ignore-path .gitignore {src,.}/{*.ts,*.js}",
-    "test": "npm run lint && npm run build"
+    "test": "yarn lint && yarn build && ava",
+    "watch:tsc": "tsc --watch",
+    "watch:ava": "ava --watch",
```

Comment:
Even if it's ava, I would put `watch:test` here ;)

Reply:
I will make a PR ^^ Usually I use `test:lint`, `test:unit`… so my main test is `run-s test:*` after it's just habits, but I would really have your feedback on this 😃